java/src/com/liferay/nativity/Constants.java

```diff
@@ -45,7 +45,7 @@
 	public static final String MENU_TEXT = "menuText";
 
 	public static final String NATIVITY_REGISTRY_KEY =
-		"SOFTWARE\\Liferay Inc\\Liferay Nativity";
+		"SOFTWARE\\Box Inc\\Box Desktop";
```

Comment:
Again, this was changed in the other PR, I feel like i'm re-reading a lot of stuff.  Did you mean to merge this with master? or the other branch?